### PR TITLE
Fix description null bug

### PR DESCRIPTION
**Pull request by asherhy**

### CODE REVIEW

**Comment by asherhy** [comment]
Fix other PED bugs mentioned in the issues

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -67,6 +68,15 @@ public CommandResult undo(Model model) throws CommandException {
                 .build();
     }
 
+    @Override
+    public boolean isAbleToRunInApplicationState(ApplicationState applicationState) {
+        if (applicationState == ApplicationState.HOME) {
```
Comment (jayasting98) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:73]: Sorry, I think this needs to be updated after my app state PR.

**Review by jayasting98 — COMMENTED** [review]
You may need to pull from master again first.

**Review by wonyeji — APPROVED** [review]
LGTM.

**Pull request merged**